TUTORIAL.md

```diff
@@ -0,0 +1,897 @@
+# OO Person
+
+![people](https://s3-us-west-2.amazonaws.com/web-dev-readme-photos/oo-labs/people.jpg)
+
+## Introduction
+
+To practice object oriented programming (OOP), you're going to create a Person class. Each instance of the `Person` class will have the ablity to:
+  - get paid/receive payments
+  - take a bath
+  - call a friend
+  - start a conversation
+  - state if they are happy and/or clean
+
+## Objectives
+1. Gain proficiency instantiating a class
+2. Gain ability to discern when to implement `attr_accessor`, `attr_reader`,  and `attr_writer`
+3. Gain abiity to discern when to define your own `attr_reader`(getter) and `attr_writer`(setter) methods
+
+# Tutorial
+## RSpec Test 1: `::new`
+
+
+```ruby
+Person
+  instantiation
+    ::new
+      a new person is instantiated with a name (FAILED - 1)
+
+Failures:
+  1) Person instantiation `::new` a new person is instantiated with a name
+     Failure/Error: let(:stella) { Person.new("Stella") }
+     NameError:
+       uninitialized constant Person
+     # ./spec/person_spec.rb:3:in `block (2 levels) in <top (required)>'
+     # ./spec/person_spec.rb:6:in `block (2 levels) in <top (required)>'
+```
+
+Our 1st hint is the NameError `uninitialized contant Person`.  This error points to the fact that we do not have a person class.
+
+Open up the person.rb file in lib directory and write the code that will get this test to pass. And then run rspec with the fail fast option, `rspec --f-f`, to get our next hint.
```

Comment:
@deniznida: oh! that's because the idea is to get them to attempt writing an answer that gets the first test to pass. If they can't (and thus they get an error) then we guide them through resolving the error.